audbcards/core/dataset.py

```diff
@@ -455,6 +750,61 @@ def create_datacard_page(dataset: Dataset):
                 fp.write('\n')
 
 
+def create_datasets_page_from_template(datasets: typing.Sequence[Dataset],
+                                       of_basename: str = 'datasets'):
+    r"""Create overview page of datasets.
+
+    Args:
+        datasets: list of datasets
+        of_basename: str
+    """
+
+    tuples = [
+        (
+            dataset.name_link,
+            dataset.short_description,
+            dataset.license_link,
+            dataset.version,
+            dataset.schemes,
+        )
+        for dataset in datasets
+    ]
+    df = pd.DataFrame.from_records(
+        tuples,
+        columns=['name', 'description', 'license', 'version', 'schemes'],
+        index='name',
+    )
+    csv_file = f'{of_basename}.csv'
+    df.to_csv(csv_file)
+
+    # Create RST file showing CSV file
+    # and adding links to all data cards
```

Comment:
The links are already in the CSV file, or are they created in addition to the CSV file (as this comment somehow implies)?

Reply:
This comment is very old and misleading. Deleted in b82a5db